integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/filterexpr/TestInFilter.scala

```diff
@@ -165,8 +168,27 @@ class TestInFilter extends QueryTest with BeforeAndAfterAll{
       Seq(Row(4, 1.00, 2.00, 3.00)))
   }
 
-  override def afterAll(): Unit = {
+  test("test infilter with date, timestamp columns") {
+    sql("create table test_table(i int, dt date, ts timestamp) stored as carbondata")
+    sql("insert into test_table select 1, '2020-03-30', '2020-03-30 10:00:00'")
+    sql("insert into test_table select 2, '2020-07-04', '2020-07-04 14:12:15'")
+    sql("insert into test_table select 3, '2020-09-23', '2020-09-23 12:30:45'")
+
+    checkAnswer(sql("select * from test_table where dt IN ('2020-03-30', '2020-09-23')"),
```

Comment:
can you add a query here with the different values in In() list as you mentioned in description?

Reply:
Testcase is already present in base